Project Tiny Url/ViewModel/TinyURLViewModel.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  TinyURLViewModel.swift
+//  Project Tiny Url
+//
+//  Created by Kurs on 13/07/2021.
+//
+
+import Foundation
+
+class TinyURLViewModel {
+    private let apiService = ApiService()
+    private var longURL = ""
+    private var tinyURLArray = [TinyURL]()
+
+    func setLongURL(longURL: String) {
+        self.longURL = longURL
+    }
+
+    // fetching data from api, saving response and loading old responses
+    func fetchDataFromApi(completion: @escaping () -> Void) {
+        apiService.performPostRequest(longURL: longURL) { [self] tinyURL in
+            // doesn't save result with no shortURL variable
+            if tinyURL.shortURL != "" {
+                saveTinyURL(tinyURL: tinyURL)
+                loadTinyURL {
+                    completion()
+                }
+            }
+        }
+    }
```

Comment:
that method indicates that it downloads the content from API, but generally it does something else.
1. it sends previously stored longURL to server and stores it in local array 
2. it decodes tinyData from UserDefaults every time after you call fetchDataFromApi method